modules/related-posts.php

```diff
@@ -6,6 +6,7 @@
  * Sort Order: 29
  * Recommendation Order: 9
  * Requires Connection: Yes
+ * Available in Development Mode: Yes
```

Comment:
Since Jetpack is moving towards a Wonderful World of Packages™, I'm thinking how we can dynamically provide more contextual information to a module/feature in a state where all it's requirements are not met. 

For example, what would it look like to have a mode header on each that lists it's requirements. The requirements could be labeled as the names of the packages themselves perhaps, or our own custom defined. 

Something like: 
```
Requires: connection, sync, error [etc...]
```

We cold then use this "Available in Development Mode" header as a separate flag, which would in turn (within each module's code), check it's requirements and if not satisfied, communicate as such during dev mode?

Reply:
Opened a discussion here p9dueE-15L-p2

Reply:
Opened a discussion here p9dueE-15L-p2